scripts/cmake/kconfig.cmake

```diff
@@ -1,9 +1,70 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
-# Kconfig targets
-
 include(${CMAKE_CURRENT_LIST_DIR}/defconfigs.cmake)
 
+
+           ### configure-time .config ###
+
+
+if(NOT INIT_CONFIG_found)
+set(INIT_CONFIG "initial.config" CACHE STRING "Initial .config file")
+
+find_file(INIT_CONFIG_found
+	NAMES ${INIT_CONFIG}
+	NO_CMAKE_FIND_ROOT_PATH
+	NO_DEFAULT_PATH
+	PATHS "." "src/arch/xtensa/configs"
```

Comment:
this doesn't seem cross platform friendly

Reply:
This list could be generated by a `for` loop in the feature. For now there are only `xtensa` and `host`. The latter has only one config file and is a bit special.

Reply:
To be honest, i still find it a bit weird that we store our defconfig there. Testbench seems to not mind it so I am ok with it.

Reply:
> To be honest, i still find it a bit weird that we store our defconfig there.

This PR will make a future search/replace easier :-)

Reply:
I added a `foreach(arch "xtensa" "host")`